src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -130,7 +131,8 @@ def reproduce(self, target_path, input_path, arguments, max_time):  # pylint: di
     syzkaller_runner = runner.get_runner(
         os.path.join(binary_dir, constants.SYZ_REPRO))
     repro_args = runner.get_config()
-    repro_args.extend(input_path)
+    repro_args.extend(
+        ['-infinite=false', '-restart_time={}s'.format(max_time), input_path])
```

Comment:
may be worth setting a slightly lowered time than `max_time` here, as `max_time` is a hard limit at which we kill the process. If we set it slightly lower in the argument here, we can give it some time to exit cleanly (if necessary).

Reply:
Is restart_time to try things multiple times ? maybe even a constant restart time is better than near the limit as process will be killed anyway. this constant can go in class constant or android/constants.py

Reply:
So experimentally, 120 seconds seems to work well. It will look for one crash and may exit before 120s is completed. I will make it as a constant as per the comment.

Reply:
Done.